google-cloud-language/google-cloud-language.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = ">= 2.0.0"
 
-  gem.add_dependency "google-gax", "~> 0.10.1"
+  gem.add_dependency "google-gax", "~> 1.0"
   gem.add_dependency "googleapis-common-protos", "~> 1.3.1"
```

Comment:
Can you also remove the googleapis-common-protos and googleauth direct dependencies from this gem as well?

Reply:
@geigerj ^

Reply:
Updated.

Reply:
LGTM